public/media-hunt/API/discover.js

```diff
@@ -0,0 +1,9 @@
+const ROOT = "/discover";
+
+export default {
+  get: {
+    discover: {
+      url: "/discover/movie",
+    },
```

Comment:
Add support for both movie and TV. Add movie and tv under discover, and have them both have their own url.

Reply:
i put root to use, and adjusted discover as per request. Hopefully its fine this time :) .

Reply:
root removed